Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderBuilderUtility.cpp

```diff
@@ -885,6 +884,36 @@ namespace AZ
                 return AZ::Success(AZStd::move(listOfFilePaths));
             }
 
+            AZStd::string GetPlatformNameFromPlatformIdentifier(const AZStd::string& platformIdentifier)
```

Comment:
Are we able to use AzFramework::OSPlatformToDefaultAssetPlatform here?

Reply:
Thanks for the tip. I looked at PlatformIdToPalFolder() but it converts "pc" to "PC" and We need "Windows".